src/Addresses.js

```diff
@@ -15,105 +17,49 @@ const CONTRACTS = {
-    USDG: "0x8deAC518eE4a4F90b89F0aFdC80d31feFdBA39F7",
-
-    StakedMycTracker: "0x07d61259194CF6760C5CdA1B91dB9B79141C25c4",
-    BonusMycTracker: "0x499084321217E00926F7bBD45830B0e74268b6cF",
-    FeeMycTracker: "0xAf3CCbfDCc5E1e66201BcCc91F5d82930E29d454",
-    StakedMlpTracker: "0x4E4dF4396EC47a7E6BDd5e731Fde9Ce108a555Bb",
-    FeeMlpTracker: "0x18dEa0cBf4f035f8733098C31D6405DfBE3F757f",
-
-    StakedMycDistributor: "0x499084321217E00926F7bBD45830B0e74268b6cF",
-    StakedMlpDistributor: "0x34964F6ad47aC4BcF2cC931839664934588Ce621",
-
-    MycVester: "0x6D97a351D4E4b34fC299d087B35f4F2f3e1Ba872",
-    MlpVester: "0xFe1775a8bd123D03f5f4588b36Bdd4CC45B5abE0",
-
-    OrderBook: "0x88Bfc6DDbF07C22416fcC6A5a9010d0a5D81d50a",
-    OrderExecutor: "0xD34beDa54de5Bc13dC592F93Be48CA7974D4bDd4",
-    OrderBookReader: "0x13d7A71851f702eD0887D71Dd55Ca4E9e401ce11",
-    PositionRouter: "0xD4aF631cEBA8C4c10a85015923F820A2ee98b119",
-    PositionManager: "0xaDDb01160aAD06cDD225770cF10c7a98502F3eEd",
-
-    ReferralStorage: "0xcDA344Dac3b6fA38245BAd1FD61900b68b3C8857",
-    ReferralReader: "0x9A3d3D5Df8F105cadb1F67587ab49d42146D5389",
-
-    FeeDistributor: "0x057B99E0112A0D9fDA7503bDc8b6995fA2ff997D",
-    FeeDistributorReader: "0x55ce0A81b697f1eff846e8a56D3f303A5BE490b1",
+    NATIVE_TOKEN: "0x08466D6683d2A39E3597500c1F17b792555FCAB9",
+    MLP: "0x25C7873FC2B2FB0A5A5D6346ee8940dCCd6A88B8",
+    TCR: "0x428F127d29f45dbB05B99a5d3adeA8171A3Ec390",
+    MYC: "0x428F127d29f45dbB05B99a5d3adeA8171A3Ec390",
+    ES_MYC: "0xe2b9C2F3c8BD2b6Bc1E75b5650133be26Ca324Af",
+    BN_MYC: "0x6cb7A4b0360Cf1B54eE9140F8Da45726671E7fdb",
+    USDG: "0x52d1A1d053CF2132C9bF9989aaaaF7921f9011BE",
+
+    StakedMycTracker: "0x45E8f0770DD204388477CfA42315E883572E0DAA",
+    BonusMycTracker: "0xa8D1b5e8b09782d64d75cd1d55444a3AbEaD0821",
+    FeeMycTracker: "0x9e958AC42115a17C6d734D02c164e761daa2BC47",
+    StakedMlpTracker: "0xFCe0334D8dd8Ae6f6dEb2784a7a6e1Bd3Dc645A2",
+    FeeMlpTracker: "0xbdf67A0a70e4A5907559f4fA784b544365A2564F",
+
+    StakedMycDistributor: "0x92E6D5464A3B4070031df73e0782951bD1a16d8C",
+    StakedMlpDistributor: "0x74F067d10D4783937Ef7d9Dc6DeF27e7489e7802",
+
+    MycVester: "0x59A3640b4C8aCF9D6E6107e9Ba16d8B7d7cC412c",
+    MlpVester: "0xd3A37697c1B21DdE4F97636E294399e0fab9D235",
+
+    OrderBook: "0xbc5E4839cb09e77650b65b7B6067d11B861C8f38",
+    OrderExecutor: "0x9B12CF1bC134a9eE01DC456fbb13a662C2dcf443",
+    OrderBookReader: "0xf2A0764144bEB244cD04356b5b2078E0ef682FF6",
+    PositionRouter: "0x2d78AE5147D358d803e6306D96db1AB995a4AAF1",
+    PositionManager: "0xb65B42c48DB28e13Cd490f72dDDEEe57eD2072f6",
+
+    ReferralStorage: "0x956BbBe040A2378B63C21011e3c71978d7B75a56",
+    ReferralReader: "0x27ec154C4D6e496027FBED863aC7a0186942Cc09",
+
+    // TODO: Change these once deployed
+    // FeeDistributor: "0x057B99E0112A0D9fDA7503bDc8b6995fA2ff997D",
+    // FeeDistributorReader: "0x55ce0A81b697f1eff846e8a56D3f303A5BE490b1",
```

Comment:
Commenting these out causes the app to crash when going to referral and rewards page. If you uncomment them it should just error without crashing the app

Reply:
Resubmitted review